alf/algorithms/actor_critic_algorithm_test.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2020 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.distributions as td
+import unittest
+
+import alf
+from alf.utils import common, dist_utils, tensor_utils
+from alf.data_structures import StepType, TimeStep
+from alf.networks import ActorDistributionNetwork, ValueNetwork
+from alf.algorithms.rl_algorithm import RLAlgorithm
+from alf.algorithms.actor_critic_algorithm import ActorCriticAlgorithm
+
+
+# adapted from alf.algorithms.rl_algorithm_test
+class MyEnv(object):
```

Comment:
Let's use the one from rl_algorithm_test directly

Reply:
Changed